tensorflow_datasets/image/celebahq.py

```diff
@@ -68,20 +68,20 @@ def __init__(self, resolution, **kwargs):
         version=v2,
         **kwargs)
     self.resolution = resolution
-    self.file_name = "data%dx%d.tar" % (resolution, resolution)
-
+    self.file_name = "data%dx%d" % (resolution, resolution)
+    self.extensions = ['.tar', '.zip', '.tar.gz', '.tgz']
```

Comment:
We should not support many format. We should only support the original datasets files from: https://github.com/tkarras/progressive_growing_of_gans.

From the README, it seems data is coming from https://drive.google.com/corp/drive/folders/0B4qLcYyJmiz0TXY1NG02bzZVRGs

Either this dataset should be updated to automatically download and apply the official pre-processing, or the pre-processed `tf_record` files should be generated using https://github.com/tkarras/progressive_growing_of_gans tools and placed inside the `manual_dir`